lib/api/2.0/obms.js

```diff
@@ -45,13 +46,56 @@ var obmsGetById = controller(function(req) {
 
 // PATCH /api/2.0/obms/identifier
 var obmsPatchById = controller( function(req) {
-    return waterline.obms.updateByIdentifier(
-        req.swagger.params.identifier.value, req.swagger.params.body.value);
+    var obmId = req.swagger.params.identifier.value;
```

Comment:
Well, the logic seems a little complicated here, suggest to put it in [obm-api-service.js](https://github.com/RackHD/on-http/blob/master/lib/services/obm-api-service.js)